Template/CSS/Site.css

```diff
@@ -10,6 +10,11 @@ body {
 
 p {
     font: 1em arial, sans-serif;
+
+    /* Restores letter‐style paragraphs. */
```

Comment:
The standard paragraph style supplied by the library now has paragraphs like a book (indented and unspaced). These three lines override that to be like a letter, the way they were before (spaced and unindented).